test/Makefile.in
=======================================================
lhs: 100644 | e10af0bdde62d0ceb4a5c28e5a387a0ab0a22e18
rhs: 100644 | ee66729f0619eff5d80831991d4f6f2dd02348f3
---@@ -84,6 +84,7 @@ DTLSV1LISTENTEST = dtlsv1listentest
 CTTEST=	ct_test
 THREADSTEST=	threadstest
 AFALGTEST= afalgtest
+D2ITEST = d2i_test
 
 TESTS=		alltests
 
@@ -106,7 +107,7 @@ EXE=	$(NPTEST)$(EXE_EXT) $(MEMLEAKTEST)$(EXE_EXT) \
 	$(CONSTTIMETEST)$(EXE_EXT) $(VERIFYEXTRATEST)$(EXE_EXT) \
 	$(CLIENTHELLOTEST)$(EXE_EXT) $(PACKETTEST)$(EXE_EXT) $(ASYNCTEST)$(EXE_EXT) \
 	$(DTLSV1LISTENTEST)$(EXE_EXT) $(CTTEST)$(EXE_EXT) $(THREADSTEST)$(EXE_EXT) \
-	$(AFALGTEST)$(EXE_EXT)
+	$(AFALGTEST)$(EXE_EXT) $(D2ITEST)$(EXE_EXT)
 
 # $(METHTEST)$(EXE_EXT)
 
@@ -124,7 +125,7 @@ OBJ=	$(NPTEST).o $(MEMLEAKTEST).o \
 	$(HEARTBEATTEST).o $(P5_CRPT2_TEST).o \
 	$(CONSTTIMETEST).o $(VERIFYEXTRATEST).o $(CLIENTHELLOTEST).o \
 	$(PACKETTEST).o $(ASYNCTEST).o $(DTLSV1LISTENTEST).o $(CTTEST).o \
-	$(THREADSTEST).o testutil.o $(AFALGTEST).o
+	$(THREADSTEST).o testutil.o $(AFALGTEST).o $(D2ITEST).o
 
 SRC=	$(NPTEST).c $(MEMLEAKTEST).c \
 	$(BNTEST).c $(ECTEST).c \
@@ -139,7 +140,7 @@ SRC=	$(NPTEST).c $(MEMLEAKTEST).c \
 	$(HEARTBEATTEST).c $(P5_CRPT2_TEST).c \
 	$(CONSTTIMETEST).c $(VERIFYEXTRATEST).c $(CLIENTHELLOTEST).c \
 	$(PACKETTEST).c $(ASYNCTEST).c $(DTLSV1LISTENTEST).c $(CTTEST).c \
-	$(THREADSTEST).c testutil.c $(AFALGTEST).c
+	$(THREADSTEST).c testutil.c $(AFALGTEST).c $(D2ITEST).c
 
 HEADER=	testutil.h
 
@@ -385,4 +386,7 @@ dummytest$(EXE_EXT): dummytest.o $(DLIBCRYPTO)
 $(AFALGTEST)$(EXE_EXT): $(AFALGTEST).o $(DLIBCRYPTO)
 	@target=$(AFALGTEST); $(BUILD_CMD)
 
+$(D2ITEST)$(EXE_EXT): $(D2ITEST).o $(DLIBCRYPTO) testutil.o
+	@target=$(D2ITEST) testutil=testutil.o; $(BUILD_CMD)
+
 # DO NOT DELETE THIS LINE -- make depend depends on it.

---